### ISSUE TITLE
[Feature Request] favorite days

### ISSUE THREAD

**Issue by blaugrau3001** [Demizo/Daily_You#94]
Not a very important feature maybe for all but for me personally and maybe for others like me. It would be cool so star days or say that some days are better than a 5/5.
Maybe you understand what I mean.

Thank you for your work btw. I really appreciate you and your work. The app is very cool and it is a part of my routine before going to bed to write a entry and have a better overview when I have a not so good day to see what I already had done.

If you need any help, maybe not from the programming side, I am not so in building android apps but for testing or something like this, it would be cool to help you there.

**Comment by Demizo** [comment]
@blaugrau3001 Good idea! One feature I've been hoping to add for a long time is tags. That way you could tag certain days and track the custom tags over time similar to moods. That feature might cover this use case.

**Comment by Demizo** [comment]
https://github.com/Demizo/Daily_You/pull/425 includes the tags feature. By default a favorite tag will be included as an example covering this use case. It can be edited or removed by the user if undesired. This should hopefully allow for much more expression when labeling or tracking events or mood.
